app/view/templates/pages/upload/step1/_form.html

```diff
@@ -139,6 +139,24 @@ <h3>Study</h3>
                 class="form-input-full form-input-blue {{ "error" if submission_form.has_error('study_name') }}"
                 placeholder="Example: https://doi.org/10.1000/182" />
         </div>
+
+        <div class="form-row">
+          <label class="full">
+            <span>Embargo date (UTC):</span>
+            <input
+                type="date"
+                min="{{ g.now.strftime("%Y-%m-%d") }}"
+                name="embargo_expires_at"
+                value="{{ submission.studyDesign['study']['embargoExpiresAt'] or '' }}"
+                class="form-input-full form-input-blue {{ "error" if submission_form.has_error('embargo_expires_at') }}" />
+
+            <div class="help">
+              Once you upload your study, you will explicitly have to publish it
```

Comment:
Consider using quotes or italics to "publish", so they understand that it is an action.
For example:

After uploading your study, you must explicitly _publish_ it to make it publicly available.
If you set an _embargo date,_ a record will appear on μGrowthDB but not your data. Instead a short message will inform visitors of the expected publication date.